test/functional/runtime/autoload/provider/python3_spec.lua

```diff
@@ -0,0 +1,93 @@
+do
+  local proc =
+    io.popen([[python3 -c 'import neovim, sys; sys.stdout.write("ok")' 2> /dev/null]])
+  if proc:read() ~= 'ok' then
+    -- Don't run these tests if python3 is not available
+    return
+  end
+end
+
+
+local helpers = require('test.functional.helpers')
+local eval, command, feed = helpers.eval, helpers.command, helpers.feed
+local eq, clear, insert = helpers.eq, helpers.clear, helpers.insert
+local expect = helpers.expect
+
+
+describe('python3 commands and functions', function()
+  before_each(function()
+    clear()
+    command('python3 import vim')
+  end)
+
+  describe('feature test', function()
```

Comment:
@Shougo can you change this describe() to it(), as well as the others. Instead of this:

```lua
  describe('feature test', function()
    it('ok')...
    end)
  end)
```

it should be this:

```lua
  it('feature test', function()
  end)
```

`describe()` is for grouping. If there's only one `it()` inside each `describe()`, it's unnecessary.